packages/plugins/Debugger/src/SNSAdaptor/components/TabContent.tsx

```diff
@@ -380,6 +383,23 @@ export function TabContent({ identity, socialAddressList }: TabContentProps) {
                             </Button>
                         </TableCell>
                     </TableRow>
+                    <TableRow>
+                        <TableCell>
+                            <Typography variant="body2" whiteSpace="nowrap">
+                                Token List
+                            </Typography>
+                        </TableCell>
+                        <TableCell>
+                            <Button
+                                size="small"
+                                onClick={async () => {
+                                    const token = await onPickToken({})
+                                    console.log(token)
```

Comment:
Remove or add some more debug info message.

```suggestion
                                    console.log('picked token', token)
```